src/server/binary/binaryServerStarter.ts

```diff
@@ -217,6 +217,45 @@ function httpHttpsGet(url: string, callback: (response: http.IncomingMessage) =>
   return http.get(url, callback);
 }
 
+/**
+ * Show a progress notification for the given get response
+ *
+ * Inspired by
+ * https://stackoverflow.com/questions/18323152/get-download-progress-in-node-js-with-request
+ *
+ * @param response the http GET response
+ */
+function showProgressForDownload(response: http.IncomingMessage) {
+  const progressOptions: ProgressOptions = {
+    location: ProgressLocation.Notification,
+    cancellable: false,
```

Comment:
why is it not cancellable?

Reply:
I can take a look into figuring out how to cancel it, I just don't know how as of right now.

Reply:
Okay, it was really straight forward